api/python/cellxgene_census/src/cellxgene_census/_testing/logger_proxy.py

```diff
@@ -0,0 +1,36 @@
+"""This module defines a plugin class that logs each request to a logfile."""
```

Comment:
It seems like py.proxy necessitates including this under the main source tree, is that correct? If so, please add a bit of explanation in this docstring (or a README.md) for posterity.

Reply:
I believe it has to be importable, so in a module that is on the python path. I don't feel super attached to having it here, so am open to other suggestions if you think there is a better way to do this.

```suggestion
"""This module defines a plugin class that logs each request to a logfile.

This class needs to be importable by the proxy server which runs in a separate process.
See the user agent tests for usage.
"""
```